### DISCUSSION TITLE
Captured Image is blank

### DISCUSSION THREAD

**Discussion by Jack-Sam** [spatie/browsershot#743] [Bugs]
Hello
I am trying to capture screenshot of a url by Spatie Browsershot. The environment is:
Ubuntu 20.04
node v14
PHP 8.0.28
puppeteer - v5.5.0
spatie/browsershot": v3.57

I could successfully get image of a sample urls, like https://google.se.
But when I try with the url of my website, it's blank.
That page is loading a map, it takes a few seconds to load fully.
So I set delay as 15 secs, but it's still blank.
Could you please help me?
Thanks in advance.

**Comment by rodolforjunior** [comment]
Hi, did you manage to solve this issue? Facing a similar problem rendering blade views with chartjs. Tried setting delay and all kinds of methods provided by the library, but no luck.

**Reply by Jack-Sam** [reply]
Did you try to set bin path?

**Reply by rodolforjunior** [reply]
No, I haven't. I ended up coming up with a different solution. Did setting the bin path resolved the issue for you?

**Reply by Jack-Sam** [reply]
Yes, disabling web security and request interception might help also.
